content/intro-to-storybook/react-native/en/get-started.md

```diff
@@ -0,0 +1,502 @@
+---
+title: 'Storybook for React Native tutorial'
+tocTitle: 'Get started'
+description: 'Setup Storybook in your development environment'
+commit: ebe2ae2
+---
+
+note that this needs to run on simulator
```

Comment:
Does this note need to be dropped?

Reply:
That note "slipped throught the cracks" (pardon the bad pun), it was removed in the last commit.